docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -161,4 +161,15 @@ Note that this is different to slow operations logged by the Couchbase SDKs - th
 *Remediation*: Review your infrastructure for signs of disk problems or any other issues. 
 Alternatively, navigate to your Data storage directory and enter the following command, making sure to fill in the correct data: `/opt/couchbase/bin/couch_dbck --verbose --json <couchbase_server_data_directory>/<bucket>/<malformed_vb_file>`, then take the response and provide it to Couchbase Technical Support.
 
+[#CB90055]
+=== Metadata Overhead (CB90055)
+
+*Background*: As Couchbase Server stores all of it's working documents in memory, if a large portion of that memory is taken up by metadata it can impact performance and force useful documents to be stored on disk rather than memory.
```

Comment:
```suggestion
*Background*: As Couchbase Server stores all of its working documents in memory, if a large portion of that memory is taken up by metadata it can impact performance and force useful documents to be stored on disk rather than memory.
```